### PR TITLE
ERMS-3140 + ERMS-3233

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: server/gokbg3/grails-app/views/apptemplates/mainTemplates/_package.gsp [hunk 1]
```diff
@@ -120,7 +120,7 @@
 
         <li>
           <a href="#ddcs" data-toggle="tab">DDCs
-            <span class="badge badge-warning">${d.ddcs?.size() ?: '0'}</span>
+            <span class="badge badge-warning">${d.ddcs.size() ?: '0'}</span>
```
Comment (philboeselager) [inline server/gokbg3/grails-app/views/apptemplates/mainTemplates/_package.gsp:123]: Why not keeping Elvis alive?

File: server/gokbg3/grails-app/views/apptemplates/mainTemplates/_tipp.gsp [hunk 2]
```diff
@@ -265,7 +265,7 @@
 
     <li>
         <a href="#ddcs" data-toggle="tab">DDCs
-            <span class="badge badge-warning">${d.ddcs?.size() ?: '0'}</span>
+            <span class="badge badge-warning">${d.ddcs.size() ?: '0'}</span>
```
Comment (philboeselager) [inline server/gokbg3/grails-app/views/apptemplates/mainTemplates/_tipp.gsp:268]: Why not keeping Elvis alive?

File: server/gokbg3/grails-app/views/apptemplates/secondTemplates/_nationalRange.gsp [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <%@ page import="de.wekb.helper.RCConstants; org.gokb.cred.RefdataCategory" %>
 <ul>
-    <g:each in="${d.nationalRanges?.sort { it.getI10n('value') }}" var="nationalRange">
+    <g:each in="${d.nationalRanges.sort { it.getI10n('value') }}" var="nationalRange">
```
Comment (philboeselager) [inline server/gokbg3/grails-app/views/apptemplates/secondTemplates/_nationalRange.gsp:3]: Why not keeping Elvis alive?

File: server/gokbg3/grails-app/views/apptemplates/secondTemplates/_regionalRange.gsp [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <%@ page import="de.wekb.helper.RCConstants; org.gokb.cred.RefdataCategory" %>
 <ul>
-    <g:each in="${d.regionalRanges?.sort { it.getI10n('value') }}" var="regionalRange">
+    <g:each in="${d.regionalRanges.sort { it.getI10n('value') }}" var="regionalRange">
```
Comment (philboeselager) [inline server/gokbg3/grails-app/views/apptemplates/secondTemplates/_regionalRange.gsp:3]: Why not keeping Elvis alive?

File: server/gokbg3/grails-app/views/public/_tippTabs.gsp [hunk 5]
```diff
@@ -42,7 +42,7 @@
 
     <li>
         <a href="#ddcs" data-toggle="tab">DDCs
-            <span class="badge badge-warning">${d.ddcs?.size() ?: '0'}</span>
+            <span class="badge badge-warning">${d.ddcs.size() ?: '0'}</span>
```
Comment (philboeselager) [inline server/gokbg3/grails-app/views/public/_tippTabs.gsp:45]: Why not keeping Elvis alive?

File: server/gokbg3/grails-app/views/tabTemplates/_showDDCs.gsp [hunk 6]
```diff
@@ -7,7 +7,7 @@
             </dt>
             <dd>
                 <ul>
-                    <g:each in="${d.ddcs?.sort { it.value }}" var="ddc">
+                    <g:each in="${d.ddcs.sort { it.value }}" var="ddc">
```
Comment (philboeselager) [inline server/gokbg3/grails-app/views/tabTemplates/_showDDCs.gsp:10]: Why not keeping Elvis alive?

**Review by philboeselager — COMMENTED** [review]
Elvis operators are a good idea if the code does not guarantee that a null value can never occur. So I would prefer to keep them in, unless there is a very good argument to drop them.

**Comment by Moetez16** [comment]
see https://github.com/hbz/laser-gokb/pull/71#discussion_r614059794. this code change was only made because of the previous remark from PR.

**Pull request merged**